README.md

```diff
@@ -3,6 +3,12 @@ Harvester Node Manager
 
 A node manager helps to manage the host kernel configuration of the [Harvester](https://github.com/harvester/harvester) cluster, eg: KSM.
 
+## Branches
+
+- `master` branch is used for development and release.
+- `v0.1.x` branch is used for Harvester v1.2.x release.
```

Comment:
Would it be possible to align the branch name with the Harvester repo in the future? This would make it more intuitive.

Reply:
I'd suggest we do this across projects in a future milestone, but still need to check if there are any potential issues of inconvenience. There are some projects like harveter-csi-dirver, cloud-provider, terraform are not bound to harvester versions.

Reply:
Sounds good. For those projects not bundled to harvester release, then individual release is reasonable.

Reply:
I created a GH issue https://github.com/harvester/harvester/issues/5559 to track this.

I will merge this PR first, and we can discuss the above issue.